opm/parser/eclipse/EclipseState/Schedule/Group.hpp

```diff
@@ -114,6 +115,7 @@ namespace Opm {
         std::shared_ptr<GroupProduction::ProductionData> m_production;
         std::shared_ptr<DynamicState<std::shared_ptr< const WellSet >> > m_wells;
         std::shared_ptr<DynamicState<bool> > m_isProductionGroup;
+        std::shared_ptr<DynamicState<bool> > m_isInjectionGroup;
```

Comment:
I don't really know - but I guess imposing value semantics on the `DynamicState` instances is a large job. Before we do an all-in conversion I guess the shared_ptr can remain here.